Packs/Netskope/Integrations/NetskopeAPIv2/NetskopeAPIv2.py

```diff
@@ -511,7 +506,7 @@ def update_url_list_command(
     url_list_id = args["url_list_id"]
     name = args.get("name")
     urls = argToList(args.get("urls"))
-    list_type = URL_LIST_TYPE[args["list_type"]] if args.get("list_type") else None
+    list_type = args.get("list_type", '').lower()
```

Comment:
```suggestion
    list_type = args.get("list_type", '').lower() or None
```

Please add a test, too, for preserving behavior